cloudbridge/providers/openstack/services.py

```diff
@@ -1249,7 +1249,7 @@ def get_or_create(self, network):
         external_nets = (n for n in self._provider.networking.networks
                          if n.external)
         for net in external_nets:
-            if self._check_fip_connectivity(network, net):
```

Comment:
We can remove the `_check_fip_connectivity` related functions entirely?

Reply:
We could, I just thought having the function might not hurt if we ever want to check for connectivity for other purposes?

Reply:
Doesn't really do anything of significance I think: https://github.com/CloudVE/cloudbridge/blob/d51933a28677e4f2cc2c077fed036c7851766145/cloudbridge/providers/openstack/services.py#L1229
Let's remove and add back if necessary.

Reply:
Done